pkgs/applications/networking/browsers/chromium/common.nix

```
@@ -749,6 +749,9 @@ let
         proprietary_codecs = true;
         enable_hangout_services_extension = true;
         ffmpeg_branding = "Chrome";
+        # enable video decode hardware acceleratation on x86_64 and aarch64
+        use_vaapi = stdenv.hostPlatform.isx86_64;
+        use_v4l2_codec = stdenv.hostPlatform.isAarch64;
```

Quick recap:
- `use_vaapi` defaults to `true` since M88
- `use_v4l2_codec` defaults to `false`

You happen to put your change into the section gated by `proprietaryCodecs`.
But to my knowledge, `proprietary_codecs`/`ffmpeg_branding` is very much independent of `use_vaapi`/`use_v4l2_codec`.

Please move them out of `proprietaryCodecs`. And while we are at it, we can also inherit the upstream defaults for `x86_64-linux` instead of repeating them (mostly style):

```suggestion
      }
      // lib.optionalAttrs stdenv.hostPlatform.isAarch64 {
        # Enable V4L2 video decoder for hardware acceleratation on aarch64:
        use_vaapi = false;
        use_v4l2_codec = true;
```